shared/api/network_forward.go

```diff
@@ -139,5 +158,9 @@ func (f *NetworkForward) Etag() []any {
 
 // Writable converts a full NetworkForward struct into a NetworkForwardPut struct (filters read-only fields).
 func (f *NetworkForward) Writable() NetworkForwardPut {
-	return f.NetworkForwardPut
+	return NetworkForwardPut{
+		Description: f.Description,
+		Config:      f.Config,
+		Ports:       f.Ports,
+	}
 }
```

Comment:
Since Put struct cannot be directly set to the Get structures anymore, should we provide `SetWritable()` function that would allow doing that?

Reply:
Sounds good